src/main.js

```diff
@@ -1,62 +1,23 @@
-import "core-js/stable";
-import "regenerator-runtime/runtime";
-import Vue from 'vue';
-import VueAnalytics from 'vue-analytics';
-import router from "./router";
-import { store } from './store/store'
-import App from './App.vue';
-import uswds from 'uswds';
-import browserDetect from 'vue-browser-detect-plugin';
-import VueMobileDetection from 'vue-mobile-detection'
-
-
-import { library } from '@fortawesome/fontawesome-svg-core';
-import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
-
-
-// social icons
-import { faTwitterSquare } from '@fortawesome/free-brands-svg-icons'
-import { faFacebookSquare } from '@fortawesome/free-brands-svg-icons'
-import { faGithub } from '@fortawesome/free-brands-svg-icons'
-import { faFlickr } from '@fortawesome/free-brands-svg-icons'
-import { faYoutubeSquare } from '@fortawesome/free-brands-svg-icons'
-import { faInstagram } from "@fortawesome/free-brands-svg-icons";
-
-Vue.component('font-awesome-icon', FontAwesomeIcon);
+import './assets/css/main.css'
 
+import { createApp } from 'vue'
+import { createPinia } from 'pinia'
+import VueUswds from "vue-uswds"
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
 
 // social icons
-library.add(faTwitterSquare);
-library.add(faFacebookSquare);
-library.add(faGithub);
-library.add(faFlickr);
-library.add(faYoutubeSquare);
-library.add(faInstagram);
-
-Vue.config.productionTip = false;
-Vue.use(uswds);
-Vue.use(browserDetect);
-Vue.use(VueMobileDetection);
-
+import { faSquareXTwitter, faFacebookSquare, faGithub, faFlickr, faYoutubeSquare, faInstagram } from "@fortawesome/free-brands-svg-icons";
+library.add(faSquareXTwitter, faFacebookSquare, faGithub, faFlickr, faYoutubeSquare, faInstagram);
 
-// create a unique ID that can be used to identify user sessions
-// Note: this section is borrowed code with Public Domain/MIT licensing
-const sessionID = function() {
-  var d = new Date().getTime();
-  if (typeof performance !== 'undefined' && typeof performance.now === 'function'){
-    d += performance.now(); //use high-precision timer if available
-  }
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
-    var r = (d + Math.random() * 16) % 16 | 0;
-    d = Math.floor(d / 16);
-    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
-  });
-};
```

Comment:
huh

Reply:
I think this was part of an earlier more elaborate approach to using GA